src/itemlistformaction.cpp

```diff
@@ -617,36 +617,21 @@ bool ItemListFormAction::process_operation(Operation op,
 		v->get_ctrl()->edit_urls_file();
 		break;
 	case OP_SELECTFILTER:
-		if (filters.size() > 0) {
+		if (filter_container.size() > 0) {
 			std::string newfilter;
-			if (automatic) {
-				if (args->size() > 0) {
-					newfilter = (*args)[0];
+			if (automatic && args->size() > 0) {
```

Comment:
This is a user-visible change too: previously, `select-filter` without arguments would be ignored in itemlist, but now it will pop up a filter selection dialog. The new behaviour is consistent with feedlist, and I think it's the right one to pick, but please add another changelog entry to document this.

Reply:
Good point, done: https://github.com/newsboat/newsboat/pull/2137/commits/91f8033498e8bb87d8f283bd74131e9df797ed67